parsers/parse_FluidModelFromGid.m

```diff
@@ -303,7 +310,41 @@
     fprintf('>> Arbitrary Lagrangian-Eulerian method selected');
 end
 
-%% 11. Load the nodes on the Neumann boundary together with the load application information
+%% 11. Load the nodes, body names and function handles for post processing
+block = regexp(fstring,'FLUID_POST_PROC_NODES','split'); 
+block(1) = [];
+out = cell(size(block));
+for k = 1:numel(block)
+    out{k} = textscan(block{k},'%f %s %s','delimiter',' ','MultipleDelimsAsOne', 1);
+end
+
+if ~isempty(out)
+    out = out{1};
+    outNodes = out(:,1);
+    nodes = outNodes{1};
+    outBodies = out(:,2);
+    bodies = string(outBodies{1}); 
+    outFctHandle = out(:,3);
+    fctHandles = string(outFctHandle{1});
+    
+    % get only the unique body names
+    postProc.nameDomain = unique(bodies)';
+    
+    % loop over the number of unique body names
+    for k=1:length(postProc.nameDomain)
```

Comment:
Leave space between the equal sign and the left/right hand side expressions, e.g. k = 1:length(postProc.nameDomain)